sources/minishell.h

```diff
@@ -22,12 +22,6 @@
 # include "resources.h"
 # include "commands/commands.h"
 
-# define FREE_ENVVARS_DLISTS 1
-# define FREE_ENVVARS_ENTRIES 2
-# define FREE_ENVVARS_KEYS 4
-# define FREE_ENVVARS_VALUES 8
-# define STAGE_FULL 15
-
 # define DBG_EXTERNAL_SEARCH_FOLDERS 1
 # define DBG_PRINT_TREE_BEFORE_EXEC 2
 # define DBG_PRINT_NODE_BEFORE_EXEC 4
```

Comment:
what for these 3 defines? thank you

Reply:
try ``./minishell --debug 1 --command MOCK_7``, ``./minishell --debug 2 --command MOCK_7``, ``./minishell --debug 4 --command MOCK_7``, ``./minishell --debug 7 --command MOCK_7``,